docs/getting_started.rst

```diff
@@ -253,6 +253,9 @@ The example below illustrates how to create a field on the BookResource that
 imports and exports its author relationships using the natural key functions
 on the Author model and modelmanager. 
 
+The resource _meta option "use_natural_foreign_keys" can enable detecting
+this setting for all Models that support it.
```

Comment:
I suggest you could be clearer on the language here:

> The resource _meta option "use_natural_foreign_keys" enables this setting for all Models that support it.

Also, please could you re-check [this paragraph](https://github.com/django-import-export/django-import-export/blob/bf4cab895df9c69173d176852dc47509748e7f76/docs/getting_started.rst#django-natural-keys).  I notice that the second sentence has a typo ( A manager class with the get_by_natural_key function is /require/required).  

Also, It might be an idea to wrap variable names / functions in double backticks, so that they are formatted in monospace.  This is done elsewhere in the documentation, but it's not always consistent.  For example:

``use_natural_foreign_keys``

Reply:
done

Reply:
Nice.  Do you mind adding the backticks to the other code elements in the 'Django Natural Keys'.  I think it would be ideal to be consistent if we can.

Reply:
I think I got them all this time, let me know if you spot any I missed